geopandas/tests/test_plotting.py

```diff
@@ -117,6 +117,12 @@ def test_single_color(self):
         ax = self.df.plot(color="green")
         _check_colors(self.N, ax.collections[0].get_facecolors(), ["green"] * self.N)
 
+        # check rgba tuple GH1178
+        ax = self.df.plot(color=(0.5, 0.5, 0.5))
```

Comment:
this is an rgb tuple, can you also try (in addition) an rgba tuple? 
(I suppose both should work the same based on the matplotlib is_color_like, but just to be explicit in the testing)

Reply:
Sure.